src/igvfd/schemas/file_set.json

```diff
@@ -72,6 +72,17 @@
                 "igvf_utils": "GEO:GSE5169185, GEO:GSE5169186, GEO:GSE5169187"
             }
         },
+        "control_type": {
+            "title": "Control Type",
+            "description": "This property describes the type of control this file-set represents.",
```

Comment:
Minor suggestions: I think this can be shortened by dropping the "This property describes". And we have not used hyphen in between "file set" previously.